ale/drivers/dawn_drivers.py

```diff
@@ -89,7 +89,7 @@ def spacecraft_name(self):
         : str
           Spacecraft name
         """
-        return self.label['INSTRUMENT_HOST_NAME']
+        return 'DAWN'
```

Comment:
Was there a reason for us not hard-coding this in the first place...?

Reply:
I'm not sure. Another thing I realized about this property is that its overwriting spacecraft_name from the pds3label class, so we don't even need this. I was just following what I've seen in other drivers, but I'm not entirely sure if we should be hardcoding this or just letting the pds3Label handle it

Reply:
We should try and use the pds3 driver when we can. INSTRUMENT_HOST_NAME should give the correct answer.

Reply:
Was the self.label (from the old code) not using the label from the pds3 mix-in before?

Reply:
Not sure if I understand the question, but the self.label reads from the label directly which from what I understand we want the mix in to be doing and not the concrete driver. Regarding the hard-coded return value, I've changed it to return the instrument_host_id property from the mix-in